app/scripts/modules/kubernetes/src/v2/manifest/manifest.service.ts

```diff
@@ -26,10 +26,14 @@ export class KubernetesManifestService {
     app: Application,
     params: IManifestParams,
     container: IManifestContainer,
+    updateCallback?: (manifest: IManifest) => void,
   ): () => void {
-    const onUpdate = (manifest: IManifest) => {
-      container.manifest = manifest || container.manifest;
-    };
+    const onUpdate =
+      updateCallback ||
```

Comment:
What's this for?

Reply:
Initially added this to support setState callbacks in React components but it looks like the subscribe is exposed directly so i'll use that instead :smile: